commands/Misc/cancertext.js

```diff
@@ -0,0 +1,105 @@
+/* eslint-disable id-length, complexity */
```

Comment:
After some discussion, we have decided that this may not be the place for this command, while no rule forbids this kind of thing, it's not something we'd like in this repository. The other pieces are fine.